application/comit_node/tests/rfc003_erc20_htlc.rs

```diff
@@ -33,7 +33,7 @@ fn given_erc20_token_should_deploy_erc20_htlc_and_fund_htlc() {
     assert_eq!(client.token_balance_of(token, alice), U256::from(1000));
     assert_eq!(client.token_balance_of(token, bob), U256::from(0));
 
-    // fund erc20 htlc
+    // Fund erc20 htlc
```

Comment:
IMO correcting the case of comments isn't worth the mess in the commit history.

Reply:
Just use `git commit --ammend`.